src/Raven.Server/Documents/Queries/IndexQueryServerSide.cs

```diff
@@ -45,6 +45,18 @@ public IndexQueryServerSide(QueryMetadata metadata)
             Metadata = metadata;
         }
 
+        public string ServerSideQuery 
+        {
+            get
+            {
+                if (Metadata.IsGraph)
+                {
+                    return Metadata.Query.GraphQuery.ToString();
+                }
+
+                return Query; //For none graph queries the Query string is identical to what the server will execute 
```

Comment:
Here I want to get the results of the query visitor, on purpose.
That give us consistent output regardless.

Reply:
i'm not sure which object you refer to.
Is this the output you want to see:
{DECLARE function __selectOutputfunction __selectOutput(__alias1, o, p, rvnQueryArgs) { 
    return {
    OrderId: id(o),
    Product: p.Name
};
}


WITH {
    FROM Orders WHERE id() = 'orders/828-A'
} AS o
WITH {
    FROM Products
} AS p
WITH...

?

Reply:
Yes, I want to strip it of as much magic as ossible.